src/daemon/http/server.rs

```diff
@@ -336,7 +334,7 @@ pub async fn health(req: Request) -> RoutingResult {
 pub async fn metrics(req: Request) -> RoutingResult {
     if req.is_get() && req.path().segment().starts_with("metrics") {
         let server = req.state();
-        let server = server.read().await;
+        let server = server;
```

Comment:
Did you keep these assignments on purpose?

Reply:
nope.. now removed. (to quick to search and remove unlocking the now removed RwLock)